control-base/src/main/java/fi/nls/oskari/control/layer/GetMapLayerGroupsHandler.java

```diff
@@ -251,6 +251,7 @@ private JSONObject getProvidersJSON(String language, Set<Integer> usedProviders)
                     }
                     JSONObject json = new JSONObject();
                     JSONHelper.putValue(json, "name", provider.getName(language));
+                    JSONHelper.putValue(json, "locale", provider.getLocale());
```

Comment:
Write the description with the requested language only/default to default language of requested is not found/default to any if default language is not found (for getDescription() implementation) .
```suggestion
                    JSONHelper.putValue(json, "locale", provider.getDescription(language));
```